pip-wheel-metadata/tutor_credentials-16.0.2.dist-info/METADATA

```diff
@@ -0,0 +1,127 @@
+Metadata-Version: 2.1
```

Comment:
I strongly suggest you create a global gitignore file on your computer and ignore `pip-wheel-metadata/` directories. This directory must not be committed.